test/Conversion/TorchToArith/basic.mlir

```diff
@@ -180,6 +180,19 @@ func.func @torch.aten.div.float(%arg0: !torch.float, %arg1: !torch.float) -> !to
   return %0 : !torch.float
 }
 
+// CHECK-LABEL:  func.func @torch.aten.div.int(
+// CHECK-SAME:                            %[[LHS:.*]]: !torch.float,
+// CHECK-SAME:                            %[[RHS:.*]]: !torch.float) -> !torch.float {
+// CHECK:          %[[LHS_F64:.*]] = torch_c.to_f64 %[[LHS]]
+// CHECK:          %[[RHS_F64:.*]] = torch_c.to_f64 %[[RHS]]
+// CHECK:          %[[SUB:.*]] = arith.divf %[[LHS_F64:.*]], [[RHS_F64:.*]] : f64
+// CHECK:          %[[OUT:.*]] = torch_c.from_f64 %[[SUB:.*]]
+// CHECK:          return %[[OUT:.*]] : !torch.float
+func.func @torch.aten.div.int(%arg0: !torch.int, %arg1: !torch.float) -> !torch.float {
```

Comment:
If you decompose the op into `AtenDivFloat` there is no need to have this test. However, you should add a corresponding test in the decompose-complex-ops.mlir file.